STYLEGUIDE.md

```diff
@@ -0,0 +1,309 @@
+
+### Don’t
+
+“Front end and full-stack developers often JavaScript.”
+
+### Do
+
+“Frontend and fullstack developers use JavaScript.”
+* * *
+
+## 9. Emojis are for personal blogs, Twitter, “flavor” content
+
+Do not use emojis. They are too casual for official docs and may not translate universally to all audiences.
+
+### Don’t
+
+“👏 Congratulations, your app is online!”
+
+### Do
+
+“Congratulations, your app is online!”
+* * *
+
+## 10. Avoid pronouns
+
+Pronouns are difficult to translate into some languages. They also tend to be gendered, which provides unnecessary bias.
+
+
+### Don’t
+
+“When he opens the app, he will see a loading screen.”
+
+### Do
+
+“When the user opens the app, it will show a loading screen.”
+* * *
+
+## 11. Avoid gerunds (-ing words)
+
+"To" forms of verbs are easier to translate.
+
+### Don’t
+
+“Setting up Amplify UI only takes two lines of code.”
+
+### Do
+
+“You can set up Amplify UI with two lines of code.”
+* * *
+
+## 12. Shorter is better
+
+Short sentences are easier to translate, for both machines and humans! Break clauses into multiple sentences.
+
+### Don’t
+
+“If you are looking to enable triggers for the Storage Category with Amazon S3 & Amazon DynamoDB as Providers, the CLI supports associating Lambda triggers with S3 and DynamoDB events.”
+
+### Do
+
+“You can enable triggers for the Storage Category with Amazon S3 & Amazon DynamoDB as Providers. Amplify supports associating Lambda triggers with S3 and DynamoDB events.”
+* * *
+
+## 13. Lead with location; end with action
+
+When instructing a learner on how to do something, start with the location of the action they’re going to perform so they can navigate to that place, then end with the action they can then use.
+
+### Don’t
+
+“Select **Edge optimized** in the **Endpoint Type** drop-down.”
+
+### Do
+
+“In the **Endpoint Type** drop-down, select **Edge optimized.”**
+* * *
+
+## 14. Use monospace fonts for code, commands, copy-ables; use bold for menus, URLs (clickable), and other UI copy
+
+Code tags and monospace fonts are used to represent code that goes into files or CLI. UI copy, file names, URLs you can click on use **bold.** Remember: monospace is for anything that lives in or is copied to a file or a command line. Everything else is bold.
+
+### Don’t
+
+“To create a new file, go to ‘Menu’ and click on `new file`.”
+
+“In your browser, open `http://localhost:3000/` ”
+
+“Enter the Amplify Library for Swift GitHub repo URL (`https://github.com/aws-amplify/amplify-swift`) into the search bar and hit **Enter**.”
+
+### Do
+
+“To create a new file, go to **Menu > New File.”**
+
+“In your browser, open **http://localhost:3000/”**
```

Comment:
A few clarifications:

- Should raw URLs be hyperlinked as well as bold?
- I'm interpreting this guidance does not apply to hyperlinked words embedded in doc content (`See [this page](https://example.com) for more information.`), but only to raw URLs instructing developers to click something or enter something.

Reply:
I have no strong opinions on this one. Do you?